src/Language/Haskell/Ghcid.hs

```diff
@@ -77,8 +83,13 @@ startGhci cmd directory echo = do
                     Just msg -> return msg
     r <- parseLoad <$> f ""
     writeIORef echo False
-    return (Ghci f,r)
 
+    let ghci = Ghci (ph, f)
+#ifndef mingw32_HOST_OS
+    tid <- myThreadId
+    installHandler keyboardSignal (Catch (interrupt ghci (Just "") True >> stopGhci ghci >> throwTo tid ExitSuccess)) Nothing
+#endif
```

Comment:
What is the installHandler doing? As far as I can tell, if you press a key in the GUI, it interrupts the running ghci process then kills the entire process. Is this an orthogonal feature? e.g. Ticket #10? Why is it calling interrupt, given exiting the parent process kills the child process anyway? Why both interrupt and stopGhci?